engine/runtime/src/main/java/org/enso/interpreter/runtime/ModuleSources.java

```diff
@@ -83,4 +86,48 @@ ModuleSources ensureSource(QualifiedName name) throws IOException {
   String getPath() {
     return file() == null ? null : file().getPath();
   }
+
+  /**
+   * Special support for reassigning locations of sources when running development version. When
+   * {@code -ea} is enabled, then we try to locate library files under the {@code
+   * distribution}/{@code lib} directories.
+   */
+  private static boolean alternativeFile(Source.SourceBuilder b, TruffleFile file) {
+    var root = file;
```

Comment:
```java
    if (!BuildVersion.ensoVersion().equals(BuildVersion.defaultDevEnsoVersion())) {
      return false;
    }
```
This returns `false` if the current version is "0.0.0-dev".

Reply:
Nice check. Used in 9596d54